standard_sequence/classy_sequence.py

```diff
@@ -747,88 +181,111 @@ def _do_mot_tof_sequence(self, t, reset_mot = False):
-        assert (shot_globals.do_molasses_img_beam or shot_globals.do_molasses_mot_beam), "either do_molasses_img_beam or do_molasses_mot_beam has to be on"
-        #TODO: what does this assert statement mean?
-        assert shot_globals.bm_ta_detuning != 0, "bright molasses detuning = 0, did you forget to correct for the new case?"
+        assert (
+            shot_globals.do_molasses_img_beam or shot_globals.do_molasses_mot_beam
+        ), "either do_molasses_img_beam or do_molasses_mot_beam has to be on"
+        assert (
+            shot_globals.bm_ta_detuning != 0
+        ), "bright molasses detuning = 0. TA detuning should be non-zero for bright molasses."
         print(f"molasses detuning is {shot_globals.bm_ta_detuning}")
 
-        #Should we do this in-sequence?
-        self.ramp_to_molasses(t, shot_globals.bm_ta_detuning, shot_globals.bm_repump_detuning)
+        _ = self.ramp_to_molasses(t)
 
         if shot_globals.do_molasses_mot_beam:
-            t = self.D2Lasers_obj.do_pulse(t, dur, ShutterConfig.MOT_FULL, shot_globals.bm_ta_power,
-                                shot_globals.bm_repump_power, close_all_shutters=close_all_shutters)
+            t, _ = self.D2Lasers_obj.do_pulse(
+                t,
+                dur,
+                ShutterConfig.MOT_FULL,
+                shot_globals.bm_ta_power,
+                shot_globals.bm_repump_power,
+                close_all_shutters=close_all_shutters,
+            )
 
         if shot_globals.do_molasses_img_beam:
-            t = self.D2Lasers_obj.do_pulse(t, dur, ShutterConfig.IMG_FULL, shot_globals.bm_ta_power,
-                                shot_globals.bm_repump_power, close_all_shutters=close_all_shutters)
+            t, _ = self.D2Lasers_obj.do_pulse(
+                t,
+                dur,
+                ShutterConfig.IMG_FULL,
+                shot_globals.bm_ta_power,
+                shot_globals.bm_repump_power,
+                close_all_shutters=close_all_shutters,
+            )
         return t
 
-    #Which arguments are actually necessary to pass or even set as a defualt?
-    #How many of them can just be set to globals?
+    # Which arguments are actually necessary to pass or even set as a defualt?
+    # How many of them can just be set to globals?
     # TODO: Maybe pass the shutter config into here? This would get rid of all the if statements?
-    def do_molasses_dipole_trap_imaging(self, t, img_ta_detuning=0, img_repump_detuning=0, img_ta_power=1, img_repump_power=1, do_repump=True, close_all_shutters=False):
-        # define quantization axis (Whut?? This is zeroing the field, there's not quantization axis)
-
-        _ = self.BField_obj.ramp_B_field(t, (0,0,0))
-
-        #Is this section needed?
+    def do_molasses_dipole_trap_imaging(
+        self,
+        t,
+        ta_power=1,
+        repump_power=1,
+        do_repump=True,
+        exposure=shot_globals.bm_exposure_time,
```

Comment:
does this argument need to be here? Why not change the exposure via the shot_globals only?